src/modelfree/train.py

```diff
@@ -384,12 +388,21 @@ def single_wrappers(single_venv, scheduler, our_idx, normalize, load_policy,
             if scheduler.is_conditional(anneal_type):
                 scheduler.set_annealer_get_logs(anneal_type, rew_shape_venv.get_logs)
 
+    if lookback_params['lb_num'] > 0:
+        lookback_venv = LookbackRewardVecWrapper(single_venv, env_name, debug,
+                                                 victim_index, victim_path, victim_type,
+                                                 transparent_params, **lookback_params)
+        single_venv = lookback_venv
+
     if normalize:
         if load_policy['type'] == 'zoo':
             raise ValueError("Trying to normalize twice. Bansal et al's Zoo agents normalize "
                              "implicitly. Please set normalize=False to disable VecNormalize.")
 
         normalized_venv = VecNormalize(single_venv)
+        if load_policy['path'] is not None:
+            normalized_venv.load_running_average(load_policy['path'].split('/')[0])
```

Comment:
What's up with the `split` here? The convention I've followed is the policy path is the directory (e.g. `final_model`). `load_running_average` looks inside there for its pickle files, and `load_stable_baselines` looks for `model.pkl`.

In any case this doesn't look right as it'd just be returning the first path component, e.g. if I have `/home/adam/...` it'd return `home`.